modules/ding_news/ding_news.pages_default.inc

```diff
@@ -1296,7 +1320,7 @@ node/%node:nid/news
   $display->title_pane = '0';
   $handler->conf['display'] = $display;
   $page->default_handlers[$handler->name] = $handler;
-  $pages['ding_news_library'] = $page;
+  $pages[''] = $page;
```

Comment:
do you mean: $pages[] = $page;? (I'm not sure how the array handles a numerical index, though...)

Reply:
This is an error caused by the features module after a field-group update.